acceptance: touch up and rename TestRaftUpdate

tbg opened the pull request:
Made the retry policy stricter since we now have Raft replay
protection. Test now logs the response streams.

Closes #7262 (as little actionable information is in that issue,
and a future repro will be better suited to determine whether it
is just a flake).

<!-- Reviewable:start -->

---

This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/7768)

<!-- Reviewable:end -->

tbg committed:
acceptance: rename Test{RaftUpdate->FreezeCluster}

tamird commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 1 of 2 files at r1, 2 of 2 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

*[acceptance/update_test.go, line 162 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/7768#-KMULTfysd3NXELsBo3q:-KMULTfzSI6rAnvYt-VM:639582638) ([raw file](https://github.com/cockroachdb/cockroach/blob/d6d2d68bc02ff781cd3bf26f797fea907d2b5909/acceptance/update_test.go#L162)):*
> ```Go
> 		defer dbStopper.Stop()
> 
> 		_, err := db.Scan(keys.LocalMax, roachpb.KeyMax, 0)
> ```

reduce scope if you eat another CI

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/7768#-:-KMULjsBFvX2HfQpKxyf:1417088721)*
<!-- Sent from Reviewable.io -->

tbg committed:
acceptance: more logging in TestRaftUpdate

Also made the retry policy stricter since we now have Raft replay
protection.

Closes #7262 (as little actionable information is in that issue,
and a future repro will be better suited to determine whether it
is just a flake).

tschottdorf commented:
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

*[acceptance/freeze_test.go, line 162 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/7768#-KMULTfysd3NXELsBo3q:-KMUM-u56wTcRPhssKst:-499146009) ([raw file](https://github.com/cockroachdb/cockroach/blob/d6d2d68bc02ff781cd3bf26f797fea907d2b5909/acceptance/freeze_test.go#L162)):*
<details><summary><i>Previously, tamird (Tamir Duberstein) wrote…</i></summary>
> reduce scope if you eat another CI
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/7768)*
<!-- Sent from Reviewable.io -->
